frontend/locales/en.json

```diff
@@ -157,7 +157,7 @@
       "issueTracker": "Found a bug? Fill an issue!",
       "sourceCode": "Read the source code",
       "versionInfo": "Version info",
-      "title": "You are using"
+      "title": "Your username"
```

Comment:
???

Reply:
Should only change the title height

Reply:
Could it be that you confused the meaning of this field?

The dialog should show the version of HedgeDoc you're currently using, for example "2.0.0". However, this version is retrieved from the backend and therefore not available in the mock-mode. To indicate this (and to not have a blank field), the dialog shows "mock" in this case.

But: This has nothing to do with the user nor its name. That the username in mock-mode is "mock" too is just coincidence.

Reply:
Thanks a lot @ErikMichelson  for making it clear for me. With real server we get the correct version number. 

![Screenshot 2023-04-20 at 4 14 57 PM](https://user-images.githubusercontent.com/26133749/233343384-7d473aa9-b1f1-4fad-9f7c-1f930e9d991e.png)

Reply:
Please change the translation to something that makes a bit more sense. Like "Running HedgeDoc Version"

Reply:
done